src/pages/index.js

```diff
@@ -4,6 +4,13 @@ import HomepageWrapper from '../global/styles';
 import ProjectModal from '../components/ProjectModal/ProjectModal';
 import getAllProjects from '../api/handlers/projects';
 
+
+export const sliceArray = (array, sliceLength) => {
+  const arrayLength = array.length;
+
+  return array.slice(0, arrayLength > sliceLength ? sliceLength : arrayLength);
+}
+
```

Comment:
this would be more appropriate as a helper function, outside of index.js which is overcrowded. Can be added as tech debt